website/docs/error-reference.md

```diff
@@ -388,6 +388,20 @@ See [5014](#5014). 5036 is the same error as [5014](#5014) but slightly modified
 to allow more common Ruby idioms to pass by in `# typed: true` (5036 is only
 reported in `# typed: strict`).
 
+## 5041
+
+Sorbet does not allow inheriting from a class which inherits from `T::Struct`.
+
+```ruby
+class S < T::Struct
+  prop :foo, Integer
+end
+
+class Bad < S; end # error
+```
+
+In the future, this error may be subsumed by [5047](#5047).
```

Comment:
This isn't very actionable for the user. Is `T::InexactStruct` a documented thing they can use instead w/ fewer guarantees, or is that a special Stripe thing that's not part of Sorbet?

Reply:
`T::InexactStruct` is part of OSS Sorbet, so we could mention it here. I think, though, that we'd eventually like to get rid of it, if possible.

If we want to provide ways to resolve this error (which, now that you mention it, I think we should), maybe we could try to avoid mentioning `T::InexactStruct` and try to guide users towards other solutions instead? cc @jez.

Reply:
Yes. I think the alternative solutions you wrote in your shipped email would be good to document here.